xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/VideoFilterShader.h

```diff
@@ -55,13 +60,23 @@ namespace Shaders {
     GLint m_hSourceTex;
     GLint m_hStepXY;
     GLint m_hStretch = 0;
+
+    GLint m_hVertex;
+    GLint m_hcoord;
+    GLint m_hProj;
+    GLint m_hModel;
+    GLint m_hAlpha;
+
+    GLfloat *m_proj;
+    GLfloat *m_model;
+    GLfloat  m_alpha;
   };
 
   class ConvolutionFilterShader : public BaseVideoFilterShader
   {
   public:
     ConvolutionFilterShader(ESCALINGMETHOD method, bool stretch, GLSLOutput *output=NULL);
-    ~ConvolutionFilterShader() override;
+    virtual ~ConvolutionFilterShader() override;
```

Comment:
clang's sanitizer will strip leading virtuals, so the next time someone does this dropping virtual will make their lives easier

Reply:
thanks, I don't know how that ended up there in the first place :)